docs/ref/templates/builtins.txt

```diff
@@ -1740,6 +1751,10 @@ with an argument of ``-1``.
     In older versions, a negative zero ``-0`` was returned for negative numbers
     which round to zero.
 
+.. versionchanged:: 3.2
+
+    The suffix ``g`` to ensure grouping by separators was added.
```

Comment:
```suggestion
    The ``g`` suffix to force grouping by separators was added.
```

Reply:
I had it that way at first and can't imagine why I changed it.